system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -760,6 +762,44 @@
                 :VariableLevel2 "CLOUD LIQUID WATER/ICE"}]
               (:ScienceKeywords (:umm concept)))))))
 
+(deftest bulk-update-update-not-found-test
+  (let [concept-ids (ingest-collection-in-each-format 
+                      find-update-keywords-umm
+                      [:dif])
+        _ (index/wait-until-indexed)
+        bulk-update-body {:concept-ids concept-ids
+                          :name "TEST NAME"
+                          :update-type "FIND_AND_UPDATE"
+                          :update-field "SCIENCE_KEYWORDS"
+                          :find-value {:Topic "DOES NOT EXIST"}
+                          :update-value {:Category "EARTH SCIENCE"
+                                         :Topic "ATMOSPHERE"
+                                         :Term "AIR QUALITY"
+                                         :VariableLevel1 "EMISSIONS"}}
+        task-id (:task-id (ingest/bulk-update-collections "PROV1" bulk-update-body))]
+      (index/wait-until-indexed)
+      (let [collection-response (ingest/bulk-update-task-status "PROV1" task-id)
+            concept-id (first concept-ids)
+            status-message (str "Collection with concept-id [" concept-id
+                                "] is not updated because no find-value found.")]
+        (is (= "COMPLETE" (:task-status collection-response)))
+        (is (= [{:concept-id concept-id,
+                 :status "FAILED",
```

Comment:
I see this is what the acceptance criteria states, but I don't think "FAILED" is the best status for the scenario where the find value is not in the metadata. I think something like "SKIPPED" would be more informative. If you agree we should discuss changing the original acceptance criteria with Kathy.

Reply:
Yes. David wrote the ticket. He is fine with "SKIPPED". Done.

Reply:
Done.